packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,455 +1,456 @@
 ## NEXT
 
-* Updates support matrix in README to indicate that iOS 11 is no longer supported.
-* Clients on versions of Flutter that still support iOS 11 can continue to use this
+- Updates support matrix in README to indicate that iOS 11 is no longer supported.
```

Comment:
Please don't edit the changlog with tooling that is configured in a way that will rewrite the style of the entire file.

Reply:
Oops, sorry for that. Will fix it right now.